pandas/core/arrays/sparse/accessor.py

```diff
@@ -369,7 +369,8 @@ def density(self) -> float:
         Ratio of non-sparse points to total (dense) data points.
         """
         tmp = np.mean([column.array.density for _, column in self._parent.items()])
-        return tmp
+        # error: Expression of type "floating" cannot be assigned to return type "float"
+        return tmp  # pyright: ignore[reportGeneralTypeIssues]
```

Comment:
Is this needed due to this PR?

Reply:
Nope. I did not do that. Came in via a merge from main.

Reply:
Could you revert any changed to this file then since this PR should be unrelated?

Reply:
Done.